jenkins_scripts/registry.py

```diff
@@ -98,7 +98,7 @@ def push(args: argparse.Namespace) -> None:
     build_parser.add_argument(
         "--registry",
         help="regsitry hostname and port",
-        default="secure.cxl-terragraph.com:443",
```

Comment:
instead of changing the defaults, we should change the github actions to pass `--registry "ghcr.io"` - that way this doesn't break the existing jobs in jenkins.